Sources/MapboxDirections/Route.swift

```diff
@@ -28,8 +31,27 @@ open class Route: DirectionsResult {
      - parameter decoder: The decoder of JSON-formatted API response data or a previously encoded `Route` object.
      */
     public required init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        tollsPrices = try container.decodeIfPresent([TollPrice].self, forKey: .tollCosts)
+        
         try super.init(from: decoder)
+        try decodeForeignMembers(notKeyedBy: CodingKeys.self, with: decoder)
+    }
+    
+    public override func encode(to encoder: Encoder) throws {
+        var container = encoder.container(keyedBy: CodingKeys.self)
+        try container.encodeIfPresent(tollsPrices, forKey: .tollCosts)
+        
+        try super.encode(to: encoder)
     }
+    
+    /**
+     :nodoc:
+     List of calculated toll costs for this route.
+     
+     See `TollPrice`.
```

Comment:
I realize this is currently `:nodoc:`, but in case we make the documentation public later, let’s mention that this property is set to `nil` unless `RouteOptions.includesTollPrices` is set to `true`.